tests/std/tests/P0896R4_ranges_alg_copy_if/test.cpp

```diff
@@ -56,64 +55,53 @@ struct not_pair {
     }
 };
 
-constexpr void smoke_test() {
-    using ranges::copy_if, ranges::copy_if_result, ranges::iterator_t;
-    using std::same_as;
-    using P = not_pair<int, int>;
+using ranges::copy_if, ranges::copy_if_result, ranges::iterator_t;
+using std::same_as;

```

Comment:
`using namespace std;` (and `const`-west) is idiomatic in modern Ranges tests.

Reply:
done